@xen-orchestra/web-core/lib/components/UiCard.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div class="ui-card" :style="{ flexDirection: direction, padding: padding ? '2.4rem' : 0 }">
```

Comment:
According to the design system, the padding is always `2.4rem`
```suggestion
  <div class="ui-card" :style="{ flexDirection: direction }">
```